assets/terraform-modules/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -37,6 +37,20 @@ systemd:
       enable: true
     - name: locksmithd.service
       mask: true
+    - name: sethostname.service
```

Comment:
I think if we add this, we should have an e2e test for it to make sure hostname on matchine matches node name.